### ISSUE TITLE
request: "Add domain" option

### ISSUE THREAD

**Issue by stibinator** [arunelias/session-alive#42]
Generally when you log into a website, you're logged into the entire domain, so it would be good if the menu had an option for adding a domain, rather than just the specific page. 

So if you're on `https://foobar.com/users/stib/?&session=123&fuz=baz&etc=etc` you could choose `Add domain` and it would add `https://foobar.com/*`.

**Comment by tsneller** [comment]
Agreed - This is necessary - It is too difficult to create multiple triggers for all of the pages you mighrt be using on a particular web site.

**Comment by arunelias** [comment]
Thank you for this feedback. This looks like a valuable enhancement to incorporate.

**Comment by MacsInSpace** [comment]
As an example, We use many instances of Jamfcloud.com. 
All of their server URLs are https://example1.jamfcloud.com, https://example2.jamfcloud.com
If it falls within the scope of this issue/request, it'd be great to be able to add https://*.jamfcloud.com too.

**Comment by arunelias** [comment]
For subdomains, the current implementation already works.

For example, I configured the Trigger URL as https://*.aws/. When I open
https://repost.aws/ and https://skillbuilder.aws/ in two separate tabs, the
two rules run independently.
